utils/languageServers.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Should this be in utils? It seems deployment specific

Reply:
(and only used by docker files)

Reply:
Yeah. This could be moved to `.deployment` but the problem is `.deployment` is in our `.dockerignore`. Not sure where to put this file.